src/components/Modal/staking-hub/DockerRunCommandModal.tsx

```
@@ -58,58 +64,260 @@ export const DockerRunCommandModal = (props: DockerRunCommandModalProps) => {
+          nodeType === 'dockerCompose' && <DockerCompose />
+        }
+        {
+          nodeType === 'binary' && <Binary />
+        }
+        {
+          nodeType === 'dappnode' && <DAppNode />
+        }
+        {
+          !!nodeType &&
+          <span
+            style={{ display: 'flex', justifyContent: 'center', marginBottom: '16px' }}
```

ESLint

[error] Expected a line break after this opening brace.

(object-curly-newline)